### PR TITLE
added load patient appointments button

### PR DESCRIPTION
**Pull request by ElijahRW**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
   * Added and connected a `Load Appointments` button to the Patient List.
   * Renamed `Search` menu tab to `Search Appointments` tab.
   * Added `Search Appointments` tab to the admin page.

### CODE REVIEW

File: client/src/app/audiologist-navigation/navigation-aids.ts [hunk 1]
```diff
@@ -91,6 +91,7 @@ export class State {
                 this.tabAvailable.add(TabsEnum.LIST_PATIENTS);
                 this.tabAvailable.add(TabsEnum.CURRENT_USERS);
                 this.tabAvailable.add(TabsEnum.LIST_PATIENTS);
```
Comment (MishalAlajmi-psu) [inline client/src/app/audiologist-navigation/navigation-aids.ts]: list patients are added twice. line 91 and line 93

**Review by MishalAlajmi-psu — CHANGES_REQUESTED** [review]

**Review by OkapalDominic — COMMENTED** [review]
Other than what Mishal said, the console.log()s should be removed before merging.
As long as you tested it on a server that has appointments, and it did, in fact, load them, then it looks good otherwise.

**Comment by gstone271** [comment]
"Load" is ambiguous of what the button actually does. Should clarify that it lists appointments.

**Comment by gstone271** [comment]
If you try to load a patient that has no appointments, it wrongly says "Patient ID not found."

**Review by OkapalDominic — APPROVED** [review]

**Review by movshov — APPROVED** [review]

**Review by wynnliam — APPROVED** [review]

**Review by MishalAlajmi-psu — APPROVED** [review]
Looks good!

**Pull request merged**